### DISCUSSION TITLE
some confusion for slider using progressbar

### DISCUSSION THREAD

**Discussion by Manvendra-div** [TomSchimansky/CustomTkinter#207] [Q&A]
in line 131 of [examples/complex_example.py](https://github.com/TomSchimansky/CustomTkinter/blob/master/examples/complex_example.py)  `                                                command=self.progressbar.set)` i can't able to find out what **value** self.progressbar.set is taking in sliders command

**Comment by felipetesc** [comment]
Hello [Manvendra-div](https://github.com/Manvendra-div). I'm sorry for being slow sometimes, but I didn't understand what seems to be the problem. 
What do you expect from [examples/complex_example.py](https://github.com/TomSchimansky/CustomTkinter/blob/master/examples/complex_example.py)
And if you provide a modified version of the sample, where the expected behavior is showned, we can try to figure it out the issue.
I'll gladly lend a hand! Thanks.

**Comment by Manvendra-div** [comment]
as i described earlier in line **131** of [examples/complex_example.py](https://github.com/TomSchimansky/CustomTkinter/blob/master/examples/complex_example.py) i found `                                                command=self.progressbar.set)` i can't able to figure out what value self.progressbar.set(**?**) is taking to work accordingly as slider.
i hope now you can able to understand

**Reply by TomSchimansky** [reply]
progressbar.set takes a single float value from 0 to 1 to set the progress bar to a specific value.

**Reply by Manvendra-div** [reply]
okhay i got that what are you saying you are saying progressbar.set(0 or 0.1 or 0.2 or so on to 1) right but my question is not this my question is i can't able to find out where CtkSlider giving values to `self.progressbar.set`

**Reply by TomSchimansky** [reply]
the command function you set to the slider with the command attribute gets called when the slider is moved and the float value of the slider is passed to the command function

**Reply by Manvendra-div** [reply]
okhay so suppose if we wanted to download file with progressbar what we can do because progressbar is taking only 0 to 1

**Comment by Valenterie** [comment]
Hey ! Had the same problem but I think i've found your issue : 
If you're looking for the output value of the slider, here's what to do : 

![code](https://user-images.githubusercontent.com/87931771/175837039-22e26d5d-f5bd-4270-8251-f186cc376702.png)

In line 3, we have the "self.output_value" that just does what you need, put that in an int variable because it will put you a 10 digits value.
